sxt/proof/inner_product/generator_fold.cc

```diff
@@ -33,6 +33,11 @@ void decompose_generator_fold(basct::span<unsigned>& res, const s25t::element& m
                               const s25t::element& m_high) noexcept {
   SXT_DEBUG_ASSERT(res.size() == s25cn::max_bits_v);
   size_t bit_index = 0;
+
+  // workaround for clang bug
+  // see https://github.com/llvm/llvm-project/issues/66568
+  volatile auto data = res.data();
```

Comment:
Well done, Ryan! 🎉 🎉

Reply:
Actually, looks like it was a bug on my part. But I put in a fix.

Reply:
Still, it was pretty cool that you managed to spot the implementation.